packages/core/src/lib/chainStreamManager/index.ts

```diff
@@ -185,10 +186,12 @@ export class ChainStreamManager {
    */
   error(error: ChainError<RunErrorCodes>) {
     if (this.finished) throw new Error('Chain already finished')
+
     this.sendEvent({
       type: ChainEventTypes.ChainError,
-      error,
+      error: new Error(error.runError ? error.runError.message : error.message),
```

Comment:
@csansoon I simplified the error returned here. We were missing the important info when is a runerror. Now we send a plain error but the error message is the good one